app/views/RoomView/index.js

```diff
@@ -336,6 +336,10 @@ class RoomView extends React.Component {
 		const observable = room.observe();
 		this.subSubscription = observable
 			.subscribe((changes) => {
+				const { navigation } = this.props;
```

Comment:
Is better if you use only `name` param, like:
```
const name = navigation.getParam('name');
if (changes.name !== name) {
  navigation.setParams({ name: changes.name });
}
```

Reply:
Ok Sir, made the changes accordingly